vhs-frontend/src/types/types.ts

```diff
@@ -1,3 +1,4 @@
+import { t } from 'i18next';
```

Comment:
Here, instead of importing the t function you should do:
`import { i18n } from '@i18n';`

That's why I previously commented that you should add an i18n export to your config.ts file.

Reply:
Then you can use i18n like so: `i18n.t(namespace:key);`, for example `i18n.t('translation:myKey');`.